### PR TITLE
Add create and edit to kustomize steps

### PR DESCRIPTION
**Pull request by shannonxtreme**
This change updates the structure of the Kustomize Manage Secrets page:

1. Update original section title to Create a secret
2. Add tabs for using string literals and using files
3. Add a section for editing an existing secret (mentioning that kustomize creates a new Secret instead of replacing the old one)
4. Update cleanup section to be a delete task

The biggest source of diff here is the addition of tabs.

This is one of three PRs that will make the "Manage Secrets using X" topics consistent to provide Create, Edit, and Delete steps for kubectl, config files, and kustomize. Original, larger PR showing the end state: #34674

This PR is part of a much larger overhaul of the Secrets documentation. The tracking issue is #4491.

/sig docs
/language en
/do-not-merge work-in-progress
/cc @tengqm

### CODE REVIEW

**Review by chrisnegus — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
This looks like a draft PR; I'm assuming that there's more work to do here. Let me know if that's wrong.

/lgtm cancel

**Comment by shannonxtreme** [comment]
Yeah I want to freshen it up and make sure the commit history makes sense. The content itself is done. I'll ping for a review when I've brought it out of draft ☺️

**Comment by sftim** [comment]
@shannonxtreme would you be willing to squash commits here? The content LGTM.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 1]
```diff
@@ -7,134 +7,122 @@ description: Creating Secret objects using kustomization.yaml file.
...
+    ```
+    secret/database-creds-5hdh7hhgfk created
+    ```
 
-```
-secret/db-user-pass-96mffmfh4k created
-```
+    When a Secret is generated, the Secret name is created by hashing
+    the Secret data and appending the hash value to the name. This ensures that
+    a new Secret is generated each time the data is modified.
```
Comment (sftim) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:86]: Please unindent this region.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md [hunk 2]
```diff
@@ -7,134 +7,122 @@ description: Creating Secret objects using kustomization.yaml file.
...
 
-## Create the Secret
+### Apply the kustomization file
 
-Apply the directory containing the `kustomization.yaml` to create the Secret.
+To create the Secret, apply the directory that contains the kustomization file:
 
-```shell
-kubectl apply -k .
-```
```
Comment (sftim) [inline content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md:71]: Did you mean to remove these lines?

**Review by sftim — COMMENTED** [review]
A quibble, I'm afraid.

File: content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md (hunk 2 above)
Comment (shannonxtreme) [reply content/en/docs/tasks/configmap-secret/managing-secret-using-kustomize.md]: The joys of early morning pre-coffee rebasing

**Comment by sftim** [comment]
Changes since [earlier LGTM](https://github.com/kubernetes/website/pull/34853#pullrequestreview-1043985646) are minor.

/lgtm
/approve

and thanks!

**Pull request merged**